client/src/components/QnAwidget.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import axios from 'axios';
+const QnAWidget = function(props) {
+  const [QList, setQlist] = React.useState([])
+  //axios call here
+
+
+  return(
+    <div>
+      <TopBar/>
+      <QuestionList list={Qlist}/>
+      <BottomBar/>
+    </div>
+  )
+}
+
```

Comment:
I think starting from line 16, you better split TopBar, QuestionList, BottomBar component into separate file (simply modularizing), because those can be reused in somewhere else.

Reply:
Split those functions into separate files.